### ISSUE TITLE
Multiple AdMob ad spaces for a single app

### ISSUE THREAD

**Issue by jahd2602** [AndlyticsProject/andlytics#669]
I can assign a single AdMob ad space to each app,  but I have some apps with multiple ad spaces. 

To reproduce :
1. Add an app 
2. In AdMob tab add an association with an ad space 
3. Try to add another ad space to the same app.  No option to do that is available.

**Comment by nelenkov** [comment]
Yes, this is the current implementation, see other open issues. If this feature is important to you, consider submitting a pull request.

**Comment by jahd2602** [comment]
Ok, I will start reading the app source code. Is there anything I need to
know before starting editing the code? Maybe you can give me a hint about
which classes I should start looking at.

**Comment by nelenkov** [comment]
Before editing anything, outline how you intend to implement it first. Stick to the current coding style, minimize diffs, and try not to break existing functionality. If your code looks good, we'll merge it. 

AdSense-related code is in the `adsense` package, see the `model` package for summaries, etc. Most UI-stuff is in the main package.

**Comment by brainvision** [comment]
I was just going to request this same feature when I read this one..
I would also appreciate very much this implementation, thanks guys!
